src/App.svelte

```diff
@@ -15,25 +15,114 @@
         playGame
 	} from './game'
 
-    let state = createInitialState()
+    let mode = 'human'
+
+    $: {
```

Comment:
what is this magic?

Reply:
reactive side-effect. it will be called whenever mode is changed